metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/File.java

```diff
@@ -123,4 +125,16 @@ public long getSize() {
     public Type getType() {
         return type;
     }
+
+    // Release date control: get the *earliest* release date from all studies which link to this file
```

Comment:
As @iyangar noted during one of the standups, the nature of transitive release date control should be documented. Will do this as well.

Reply:
Documentation added.

Reply:
Could you please convert it to Javadoc? It basically describes what the field is. Applies to other classes.

Reply:
Good suggestion, fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/85037d329878a7903955a76d5efd5842287cf87b.